src/electron/flux/store/android-setup-store.ts

```diff
@@ -30,7 +30,7 @@ export class AndroidSetupStore extends BaseStoreImpl<AndroidSetupStoreData> {
         // the value of currentStepId below is not especially meaningful
         // because the state will be overridden on the call to initialize
         // when the state machine factory is created.
-        return { currentStepId: 'detect-adb' };
+        return { currentStepId: 'prompt-choose-device' };
```

Comment:
to be reverted

Reply:
will it be reverted as part of this PR or after?

Reply:
as part